scripting/executes.sp

```diff
@@ -338,7 +337,6 @@ public void OnPluginStart() {
   }
 
   g_AllowAWPCookie = RegClientCookie("executes_awpchoice", "", CookieAccess_Private);
-  g_SilencedM4Cookie = RegClientCookie("executes_silencedm4", "", CookieAccess_Private);
   g_CZCTSideCookie = RegClientCookie("executes_cz_ct_side", "", CookieAccess_Private);
```

Comment:
I'm not sure what the implications of using cookies without ever registering them with `RegClientCookie` is. I'd rather continue to register the new ones as well, unless there's some documentation in SM around what happens here.